### DISCUSSION TITLE
Beste Hardware (inkl. Funk) für Raspberrymatic + Homeassistent

### DISCUSSION THREAD

**Discussion by ck873** [OpenCCU/OpenCCU#2807] [🇩🇪 Generelle Diskussionen]
Hallo zusammen,

ich möchte neben meinem Raspberrymatic Homeassistent nutzen und stehe vor der Entscheidung welche Hardware dazu am sinnvollsten ist. Für mein Raspberrymatic nutze ich aktuell eine Thinkerboard S (mit RPI-RF-MO Funkmodul) und bin damit sehr zufrieden (sowohl Leistung als auch Funkreichweite sind absolut ausreichend). Für mein Homeassistent habe ich mir einen Dell Wyse 5070 Mini PC angeschafft (unter Proxmox sollen dann zusätzlich noch OMV und Pi-Hole laufen). Nun gäbe ich es folgende Varianten und ich bin mir nicht sicher welches davon die sinnvollste ist:
1) Raspberrymatic (Thinkerboard S mit RPI-RF-MO Funkmodul) und Dell Wyse 5070 mit Homeassistent VM unter Proxmox
2) Dell Wyse 5070 (HmIP-RF-USB) mit Homeassistent VM und Raspberrymatic Add-on unter Proxmox
3) Dell Wyse 5070 (HmIP-RF-USB) mit Homeassistent VM und seperater Raspberrymatic VM unter Proxmox

Wichtig ist mir, dass:
a) alle Entitäten von Raspberrymatic unter Homeassistent genutzt werden können
b) das Gesamtsystem stabil läuft 
c) der Stromverbrauch möglichst gering ist
d) die Funkstabilität möglichst groß sein soll
e) das System möglichst zukunftssicher ist

Bekannte Nachteile:
- Raspberrymatic + Dell Wyse 5070 = ~6 + ~8  = ~14Watt
- Störungen mit HmIP-RF-USB Funkmodul je nach Ausstellungsort möglich
- Homeassistent VM und Raspberrymatic Add-on scheinbar nicht so stabil

Was sind eure Erfahrungen, bzw. welche Hardware haltet ihr für ideal

**Comment by Baxxy13** [comment]
Also ich würde 3. machen.
Den Dell Wyse 5070 Mini PC gibt's ja in verschiedenen Ausstattungen, da wäre mal interessant zu wissen welche CPU und wieviel RAM deiner hat. 
Eines meiner Hauptsysteme ist auch so ein Mini-PC mit Celeron J4125 und 8GB RAM. Da läuft eine RaspberryMatic-VM (2GB-RAM) und eine Homeassistant-VM (4GB RAM). 1GB nimmt schon Proxmox so das da nicht mehr viel "RAM-Luft" ist. 

Das RaspberryMatic AddOn innerhalb einer Homeassistant-VM solltest du möglichst vermeiden. "Doppel-Virtualisierung" ist nicht gut.

**Comment by ck873** [comment]
Auf meinem Wyse 5070 habe ich 16GB Ram und den J5005 CPU, von daher besteht keine Ressourcenknappheit. Hat sonst noch jemand einen Tipp? Speziell die Funkstabilität und Reichweite von Variante 3 würde mich sehr interessieren!

**Reply by Baxxy13** [reply]
> Funkstabilität und Reichweite

Na das hat quasi nichts mit der eingesetzten Hardware (auf der RaspberryMatic läuft) zu tun. 
Hier kommt's eher auf das Funkmodul und dessen Positionierung oder auf die Positionierung einer externen Antenne (am Funkmodul) an.

Mal als Beispiel:
Du hast einen HmIP-RFUSB irgendwo positioniert.
Ob der nun an einem Pi2B oder einem Celeron J5005 hängt spielt keine Rolle.  

Da eins meiner Systeme im Ferienhaus residiert und ich da nur selten vor Ort bin läuft das System schon mal mehrere Monate ohne Interaktion meinerseits durch.  (aktuell seit 09.05.24)

**Comment by wtremmel** [comment]
Nachdem ich jahrelang RaspberryMatic und HomeAssistant auf zwei Devices betrieben habe, habe ich nun umgestellt auf:

- HomeAssistant läuft (wie bisher) auf HomeAssistant Yellow mit RaspberryPI 4 CM
- RaspberryMatic läuft als AddOn

Der Yellow hat zwei USB-Ports:
- einer belegt mit einem ZigBee Stick
- der andere mit HomeMatic HmIP-RFUSB

Als Integration in HomeAssistant hatte ich bis vor kurzem die "eingebaute" wie [hier](https://www.home-assistant.io/integrations/homematic/) beschrieben, jetzt umgestellt auf die [Integration aus HACS](https://github.com/danielperna84/custom_homematic).

Bis jetzt funktioniert alles prima, bei der Umstellung musste ich allerdings alle Homematic-IP Geräte einmal zurücksetzen (aber das kann andere Gründe haben, ich hatte schon vorher Probleme).
